mapclientplugins/fieldworkgait2392geomstep/__init__.py

```diff
@@ -1,12 +1,10 @@
-
-'''
+"""
 MAP Client Plugin
-'''
+"""
 __version__ = '0.1.2'
```

Comment:
Can you change this version to 0.2.0 please.

Reply:
Done.